app.py

```diff
@@ -1,24 +1,24 @@
 from flask import Flask, render_template
 import os
 import simplejson as json
```

Comment:
Sort and format the imports as follows: http://google.github.io/styleguide/pyguide.html#313-imports-formatting

Reply:
Ready! [564fbce](See https://github.com/fund3/sentiment-dashboard/pull/3/commits/564fbce1529837aff9d8f5d1000d3bb72073b28b)